go.mod

```diff
@@ -99,7 +99,8 @@ replace (
 	github.com/go-logr/logr => github.com/go-logr/logr v0.2.1-0.20200730175230-ee2de8da5be6
 	github.com/go-logr/zapr => github.com/go-logr/zapr v0.2.0
 	// pin to prevent panics in DeepEqual() when comparing envoy objects
```

Comment:
I would remove the entire comment as we are now using `reflect.DeepEqual` and this no longer applies.

Reply:
Good point, it's fixed now.